### PR TITLE
kakakumag.com  

### PR DESCRIPTION
**Pull request by SKEIDs**
website URL : https://kakakumag.com/houseware/?id=17763

<details><summary>Screenshot:</summary>

image
</details><br/>

### CODE REVIEW

File: AnnoyancesFilter/sections/self-promo.txt [hunk 1]
```diff
@@ -214,6 +214,9 @@ analyticsvidhya.com##a[id^="stickyban"]
 testbook.com##.post-body-banner a[href="https://testbook.com/pass"] > img
 ! promotional ads
 kakakumag.com##.side-ad-box
+kakakumag.com###_popIn_recommend
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/self-promo.txt:217]: It seems all these 3 rules changes nothing

File: SocialFilter/sections/specific.txt [hunk 2]
```diff
@@ -47,6 +47,7 @@ originalnews.nico,archives.gov,sportnews.to,nationalchronicle.in,analogictips.co
 sedaily.com,canshop.jp,koreatimes.co.kr,m.chosun.com,sports.donga.com##.sns_area
 !
 dekiru.net##.module-individual-article-item-share
+kakakumag.com##.article-sns-btm
```
Comment (Sergey-Lyapin) [inline SocialFilter/sections/specific.txt:50]: There is the same block at the top - so it's better to use `kakakumag.com##div[class^="article-sns-"]`

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/self-promo.txt (hunk 1 above)
Comment (Yuki2718) [reply AnnoyancesFilter/sections/self-promo.txt:217]: Also note popIn is fully blocked only if TPF is enabled, so this kind of filter should go on Spyware > specific.

Comment (Yuki2718) [reply AnnoyancesFilter/sections/self-promo.txt:217]: > It seems all these 3 rules changes nothing

The reason is that `.container-side-wrap` has fixed height. I don't want to override this just to remove small placeholder at the bottom.

Comment (SKEIDs) [reply AnnoyancesFilter/sections/self-promo.txt]: ok. I delete these 3 rules

File: SocialFilter/sections/specific.txt (hunk 2 above)
Comment (SKEIDs) [reply SocialFilter/sections/specific.txt]: ok

**Review by Yuki2718 — APPROVED** [review]

File: AnnoyancesFilter/sections/self-promo.txt (hunk 1 above)
Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/self-promo.txt]: @SKEIDs Without tracking protection filter the links in the popin widget lead to site's another articles - that's why we don't remove this widget in adblock filters.

**Pull request merged**